keys/generate-key-pair.sh

```diff
@@ -7,13 +7,6 @@ Usage: ${0##*/} [<encryption-keyType>]
 
 Arguments:
   <encryption-keyType>  The type of encryption to use when generating owner key pair (ecdsa256, ecdsa384, rsa, or all). Will default to all.
-
-Required environment variables:
```

Comment:
I would like you to have optional env vars for each input value, and then below only prompt for the values that were not specified in env vars.